assets/css/global.css

```diff
@@ -54,3 +54,27 @@ main {
     @apply grid grid-cols-1 lg:grid-cols-2 place-items-center  items-start lg:gap-x-14 gap-y-8;
   }
 }
+
+.article {
+  h1, h2, h3, h4, h5, h6 {
+    @apply my-4;
+    a {
+      @apply !font-LTZarid;
+    }
+  }
+  a {
+    font-size: unset;
+  }
+  p, ul, ol, li {
+    @apply text-2xl my-1;
+  }
+}
+ul {
```

Comment:
Why is this a generic class classification?

Reply:
Because the same styling would be applied to methodology or findings articles, and tailwind resets styling by default, so I had to add the basic styling to the element.